LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 CHAOSS
+Copyright (c) 2019 CHAOSS
```

Comment:
Please make pull requests for one thing at a time. 

For example, do not change the license file, when the pull request is about updating the README.

Reply:
Sure @GeorgLink, I would make a new PR for the license file including the previous year!

Reply:
Sorry about this @GeorgLink, will make a seperate PR for this change.

Reply:
Thanks @harshalmittal4 
With pull request #135, the LICENSE file change can be removed form pull request #132 (this pr)

Reply:
Yes, @GeorgLink, but I first need to resolve the conversation corresponding to the Maintainers/Contributors to update this PR. Please update me on it :)